README.md

```diff
@@ -1,42 +1,51 @@
 # Exercism Swift Test Runner
```

Comment:
I've updated this file to make it similar to existing test runner READMEs, which was needed because I change the test script files to be consistent too.